### PR TITLE
Acknowledgements update

### PR DESCRIPTION
**Pull request by chaals**

### CODE REVIEW

**Comment by chaals** [comment]
One reviewer should be enough. I merged the 5.2 acknowledgements into the general 5.x historical ones, moved former editors, and tried to give more consistent credit to groups whose work we have included. As always, still more to do...

File: sections/acknowledgements.include [hunk 1]
```diff
@@ -974,30 +912,43 @@
...
   Šime Vidas, <!-- again in 5.1 -->
-  Simo Sutela,
   Simon Montagu,
+  Simon Pieters,
   Simon Spiegel,
+  Simo Sutela,
   skeww, <!-- on reddit -->
+  S. Mike Dierken,
   Smylers,
+  <span lang="tr">Tantek Çelik</span>,
```
Comment (chaals) [inline sections/acknowledgements.include:944]: move to the right place in the order

File: sections/acknowledgements.include [hunk 2]
```diff
@@ -881,8 +804,11 @@
   Olli Pettay,
   oSand, <!-- from reddit -->
   Pablo Flouret,
+  Pankit Gami,
+  Patrick Dark,
   Patrick Garies,
   Patrick H. Lauke,
+  Patrick Lauke,
```
Comment (chaals) [inline sections/acknowledgements.include:813]: duplicate

File: sections/acknowledgements.include [hunk 3]
```diff
@@ -828,42 +733,60 @@
...
   Nadia Heninger,
-  Nhan,
+  "namitos", <!-- https://github.com/namitos-->
   NARUSE Yui,
+  Nathan Lorberger,
   Neil Deakin,
   Neil Rashbrook,
   Neil Soiffer,
+  Neil "ww3", <!-- https://github.com/ww3 -->
+  Nhan,
```
Comment (chaals) [inline sections/acknowledgements.include:780]: duplicate

File: sections/acknowledgements.include [hunk 4]
```diff
@@ -828,42 +733,60 @@
...
   Mihai Parparita,
+  Mihai Şucan<!-- from ROBO Design -->,
+  "Mijabi", <!-- https://github.com/mijabi-->
   Mike Brown,
   Mike Dierken<!-- S. Mike Dierken -->,
   Mike Dixon,
   Mike Hearn,
   Mike Schinkel,
   Mike Shaver,
+  Mike™ Smith,
```
Comment (chaals) [inline sections/acknowledgements.include:758]: duplicate with michael(tm)

File: sections/acknowledgements.include [hunk 5]
```diff
@@ -619,47 +489,59 @@
...
+  Jason Edelman,
   Jason Kersey,
   Jason Kiss,
   Jason Lustig,
   Jason White,
   Jasper Bryant-Greene,
   Jasper St. Pierre,
   Jatinder Mann,
+  J.C. Jones,
+  "jdsmith3000", <!-- https://github.com/jdsmith3000-->
```
Comment (chaals) [inline sections/acknowledgements.include:512]: duplicate

File: sections/acknowledgements.include [hunk 6]
```diff
@@ -1,288 +1,69 @@
...
   Thanks to the participants of the
-  <a href="https://www.w3.org/community/respimg/participants">Responsive Images Community Group</a>
-  and the WHATWG for helping to develop the <{picture}> element, the <{img/srcset}> attribute, and
-  the <{source/sizes}> attribute. Special thanks to Bruce Lawson for originally suggesting, Edward
-  O'Connor and Ian Hickson for writing the original srcset specification, and Adrian Bateman for
-  providing the group with guidance. Contributions also from: David Newton, Ilya Grigorik, John
-  Schoenick, and Leon de Rijke.
-
+  the <a href="">Accessible Platform Architecture Working Group</a>
+  the <a href="">Internationalization (i18)) Working Group</a>
```
Comment (chaals) [inline sections/acknowledgements.include:55]: remove extra right paren ")"

**Review by chaals — COMMENTED** [review]
Need to fix a few duplicate entries and oddities.

Should also check acknowledged *groups*

**Comment by edent** [comment]
I'm confused by why some people have a URl wrapped in a comment tag. Should they be linked - or are they just there for reference?

**Comment by chaals** [comment]
It's because they are not "real names", but a pointer for reference.

Random selections of them were linked in the past. An alternative would be to offer to link _any_ name. But the spec already has more links than it needs :)

File: sections/acknowledgements.include [hunk 7]
```diff
@@ -485,106 +317,139 @@
...
   Doug Simpkinson,
+  Drew McLellan,
   Drew Wilson,
+  "duckware", <!-- https://github.com/duckware-->
   Dylan Barrell,
   Edmund Lai,
-  Eduard Pascual,
   Eduardo Vela,
+  Eduard Pascual,
   Edward O'Connor,
```
Comment (hober) [inline sections/acknowledgements.include]: You can drop this line.

Comment (chaals) [reply sections/acknowledgements.include]: yup. (Sorry, thought that had been done already)

**Comment by chaals** [comment]
Added acks for #1027, resolved conflicts.

**Comment by bkevans98** [comment]
Pulls

**Pull request merged**